eox_core/tests/integration/README.rst

```diff
@@ -1,429 +0,0 @@
-Integration tests
```

Comment:
If these integration tests were being executed somewhere (Jenkins?), dropping them would require a breaking change bump. I don't think it is, but we'll have to check either way.

Reply:
I agree with you, I didn't want to delete them either. How can we verify what you suggest? Because eliminating this module is part of the acceptance criteria for this task. @mariajgrimaldi :eyes:

Reply:
The job executing this test was called `Saas-PROD-QA-eox-core`, and according to an internal spreadsheet, it was marked as to be deleted. Do you have access to our old Jenkins installation? If you do, could you check if it was deleted? In any case, I don't think this would merit a breaking change since the last time the job was successfully executed was 3 years ago.

Reply:
@mariajgrimaldi, so can we remove the `eox_core/tests/integration` module?